services/types.go

```diff
@@ -3,4 +3,5 @@ package services
 type OnboardingTicket struct {
 	ID             string `json:"id"`
 	ExpirationDate int64  `json:"expirationDate,string"`
+	StorageQuota   *int   `json:"storageQuota,omitempty"`
```

Comment:
The content of the token was designed to be viewable by the end user. Let's be more specific with the name to remove any kind of ambiguity 
```suggestion
	StorageQuotaInGiB   *int   `json:"storageQuotaInGiB,omitempty"`
```